trueos-handbook/introducing.rst

```diff
@@ -909,30 +1013,30 @@ frame to access the storage screen seen in :numref:`Figure %s <vbox10>`.
 
    Virtual Machine Storage Settings
 
-Double-click the word :guilabel:`Empty`, which represents the DVD
-reader. To access the |trueos| installer from the DVD reader,
-double-check the :guilabel:`Slot` is pointing to the correct location
-(e.g. :guilabel:`IDE Secondary Master`) and use the drop-down menu to
-change it if the location is incorrect.
+Click the word :guilabel:`Empty`, which represents the DVD reader. To
+access the |trueos| installer from the DVD reader, double-check the
+:guilabel:`Slot` is pointing to the correct location (e.g.
+:guilabel:`IDE Secondary Master`) and use the drop-down menu to change
+the location if incorrect.
 
 If using an ISO stored on the hard disk is preferred, click the
 :guilabel:`DVD` icon then :guilabel:`Choose a virtual CD/DVD disk file`
 to open a browser menu to navigate to the location of the ISO. Highlight
 the desired ISO and click :guilabel:`Open`. The name of the ISO will now
 appear in the :guilabel:`Storage Tree` section.
 
-|trueos| is now ready to be installed into the virtual machine.
-Highlight the virtual machine and click on the green :guilabel:`Start`
-icon. A window opens, indicating the virtual machine is starting. If a
-DVD is inserted, it should audibly spin and the machine will start to
-boot into the installation program. If it does not or if using an ISO
-stored on the hard disk, press :kbd:`F12` to select the boot device
-when the message to do so appears, then press :kbd:`c` to boot from
-CD-ROM. Proceed through the installation as described in the
-:ref:`Install` section.
+|trueos| is now ready to be installed into the virtual machine as a
+guest OS. Highlight the virtual machine and click on the green
+:guilabel:`Start` icon. A new window opens, indicating the virtual
+machine is starting. If a DVD is inserted, it should audibly spin and
+the machine will start to boot into the installation program. If it does
+not or if using an ISO stored on the hard disk, press :kbd:`F12` to
+select the boot device when the message to do so appears, then press
+:kbd:`c` to boot from CD-ROM. Proceed through the installation as
+     described in the :ref:`Install` section.
```

Comment:
Extra whitespace here?

Reply:
ahh yeah, that's the editor trying to be smart with indenting.